Updated more meta file and added some tracks

SonicRPika opened the pull request:
Added the following:

Sonic the Hedgehog 2 (8 bit) - Green Hills Zone
Sonic Chaos - Mecha Green Hill Zone
Sonic CD (JP) - You Can Do Anything
Pokémon Scarlet/Violet - Battle Penny

Also updated some of the Sonic tracks I have made to have groups. Haven't done them all yet though

TheSoundDefense reviewed (commented):
This PR adds a second "Endless Possibility" song with the exact same title as the first. This breaks the randomizer. Please double check this.

SonicRPika commented:
Hope this fixes this

TheSoundDefense commented on data/Music/Pokemon Series/Scarlet and Violet/Battle! vs Penny.meta:
```diff
@@ -0,0 +1,4 @@
+Pokémon Scarlet/Violet - Battle! vs Penny
```
This song is close to the line of being too loud. You might want to bring the volume down just a bit.

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic 2 (8 bit)/Green Hills Zone.meta:
```diff
@@ -0,0 +1,4 @@
+Sonic the Hedgehog 2 (8 bit) - Green Hills Zone
+0x03
+bgm
+HyruleField,Fields
```
The location line should have at least one group from each level of specificity (exact, high, mid and low). We can merge the PR without this, but then someone else will come in and add the missing groups for you, just so you know.

TheSoundDefense replied:
```suggestion
HyruleField,Fields,Outdoors,Overworld
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta:
```diff
@@ -0,0 +1,4 @@
+Sonic CD (JP) - You Can Do Anything
+0x21
+bgm
+TitleTheme
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta:
```diff
@@ -0,0 +1,4 @@
+Sonic CD (JP) - You Can Do Anything
```
A couple of notes:
- The Malon voice is kind of buried in the mix, especially near the end. You might want to bring this out more.
- Having the entire song come to an end and then start again feels strange. I would recommend finding another loop point so the music is not constantly interrupted.

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic CD/jpbadstardustspeedway.meta:
```diff
@@ -1,3 +1,4 @@
-bgm
\ No newline at end of file
+bgm
+BigFight,DampeRace
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).

TheSoundDefense replied:
```suggestion
BossBattle,FireBoss,DodongosCavern,BossFight,BigFight,Fight
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic Chaos/Mecha Green Hill Zone.meta:
```diff
@@ -0,0 +1,4 @@
+Sonic Chaos - Mecha Green Hill Zone
+0x03
+bgm
+HyruleField,Fields,WindmillHut
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).

TheSoundDefense replied:
```suggestion
HyruleField,LostWoods,GerudoValley,WindmillHut,Fields,Outdoors,Overworld
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic Rush/VelaNova.meta:
```diff
@@ -1,3 +1,4 @@
-bgm
\ No newline at end of file
+bgm
+GanonBattle,GanondorfBattle
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).

TheSoundDefense replied:
```suggestion
GanondorfBattle,GanonBattle,FinalFight,BigFight,Fight
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/Fanfare - 1Up.meta:
```diff
@@ -1,3 +1,4 @@
-Fanfare
\ No newline at end of file
+Fanfare
+HeartContainerGet,HeartPieceGet,EventFanfare
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).
```suggestion
HeartContainerGet,HeartPieceGet,ItemFanfare,EventFanfare
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/MarbleZone.meta:
```diff
@@ -1,3 +1,4 @@
-bgm
\ No newline at end of file
+bgm
+DodongosCavern,FireTemple,Dungeon,AncientDungeon
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).

TheSoundDefense replied:
```suggestion
DodongosCavern,FireTemple,ChildDungeon,AncientDungeon,AdultDungeon,Dungeon
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta:
```diff
@@ -1,3 +1,2 @@
-Sonic the Hedgehog - Final Zone
-0x03
-bgm
```
Were the changes to this file intended?

SonicRPika replied:
I either forgot to update it, or didn't have a good location at the time and forgot to remove it. I have fixed this

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap1.meta:
```diff
@@ -1,3 +1,4 @@
-bgm
\ No newline at end of file
+bgm
+IceCavern,ZorasDomain
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).

TheSoundDefense replied:
```suggestion
ZorasDomain,IceCavern,MysticalDungeon,AdultDungeon,Dungeon
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap2.meta:
```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog 3 & Knuckles - Ice Cap Act 2
 0x03
 bgm
+IceCavern,ZorasDomain
```
The location line should have at least one group from each level of specificity (exact, high, mid and low).

TheSoundDefense replied:
```suggestion
ZorasDomain,IceCavern,MysticalDungeon,AdultDungeon,Dungeon
```

TheSoundDefense reviewed (commented):
I added some comments that you don't have to address to merge this PR, but I'd like you to take a look at them.

SonicRPika commented:
Addressing most of the comments here:

If I haven't given a location of each specificity, it's because I got confused when looking at other meta files. I am happy for those with some added already to get more added to it (most likely whichever one groups the most added).
I have fixed the sound concern of Battle Penny by reducing the total volume by 8. Hopefully this will help
Fixed my error with one of the meta files as I must have uploaded on accident.

With the You Can Do Anything item, it's technically a song/intro so there isn't really a good loop point for it (same for tracks like Team Fortress 2 - Right Behind You), however I have set all non-vocal parts of the song to be lower with Malon being at max volume

TheSoundDefense commented:
I'll fix up the location tags for your songs before merging this.

For the future, I wrote a web tool that should take a lot of the confusion out of location tagging. You can try it out [here](https://thesounddefense.github.io/musicgroups/).

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta:
```diff
@@ -1,3 +1,4 @@
-Sonic the Hedgehog - Final Zone
+Sonic the Hedgehog 1 - Final Zone
```
```suggestion
Sonic the Hedgehog - Final Zone

```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta:
```diff
@@ -1,3 +1,5 @@
-bgm
\ No newline at end of file
+bgm
+EndFight,BossBattle,FireBoss,GanondorfBattle,GanonBattle
```
```suggestion
GanondorfBattle,GanonBattle,FinalFight,BigFight,Fight
```

TheSoundDefense commented on data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/Sando2.meta:
```diff
@@ -1,3 +1,4 @@
-bgm
\ No newline at end of file
+bgm
+SpiritTemple,CastleUnderground,ShadowTemple,ForestTemple,Dungeon,AdultDungeon,MysticalDungeon
```
```suggestion
ForestTemple,SpiritTemple,ShadowTemple,CastleUnderground,AncientDungeon,MysticalDungeon,SpookyDungeon,AdultDungeon,Dungeon
```
